boards/nxp/mimxrt1060_evk/mimxrt1060_evk.dts

```diff
@@ -186,6 +186,30 @@ zephyr_udc0: &usb1 {
 	status = "okay";
 };
 
+zephyr_uhc0: &uhc1 {
+	status = "okay";
+	phy_handle = <&usbphy1>;
+};
+
+zephyr_uhc1: &uhc2 {
+	status = "okay";
+	phy_handle = <&usbphy2>;
+};
+
+&usbphy1 {
+	status = "okay";
+	D_CAL = <12>;
```

Comment:
We should use property names here that are lowercase with dashes, like `d-cal`. This aligns with the [Linux devicetree coding style](https://docs.kernel.org/devicetree/bindings/dts-coding-style.html#naming-and-valid-characters)